src/components/PageElements/Elements/HistoryResponseElement.tsx

```diff
@@ -15,41 +16,50 @@ const HistoryResponseElement: React.FC<HistoryElement> = ({
   elementTypePluginCode,
   parameters,
 }) => {
+  const { strings } = useLanguageProvider()
   return (
     <div className="response-element-content">
       <p className="secondary author-name">{author}</p>
       <div className="decision-container">
-        <div className="comment-container">
-          <Header
-            as="h5"
-            // icon={attentionRequired && <Icon name="exclamation circle" color="red" size="tiny" />}
-            content={title}
-          />
-          <p className="secondary date-indicator">{getSimplifiedTimeDifference(timeUpdated)}</p>
-        </div>
-        {elementTypePluginCode ? (
-          <SummaryViewWrapper
-            element={
-              {
-                parameters,
-                pluginCode: elementTypePluginCode,
-                isRequired: true,
-                isVisible: true,
-              } as ElementState
-            }
-            response={response}
-            // Not passing any other reponses, as these elements will be using
-            // pre-evaluated paramaters, not doing new evaluations
-            allResponses={{}}
-          />
+        {message ? (
+          <>
+            <div className="comment-container">
+              <Header
+                as="h5"
+                // icon={attentionRequired && <Icon name="exclamation circle" color="red" size="tiny" />}
+                content={title}
+              />
+              <p className="secondary date-indicator">{getSimplifiedTimeDifference(timeUpdated)}</p>
+            </div>
+            {elementTypePluginCode && response.evaluatedParameters ? (
+              <SummaryViewWrapper
+                element={
+                  {
+                    parameters,
+                    pluginCode: elementTypePluginCode,
+                    isRequired: true,
+                    isVisible: true,
+                  } as ElementState
+                }
+                response={response}
+                // Not passing any other responses, as these elements will be using
+                // pre-evaluated parameters, not doing new evaluations
+                allResponses={{}}
+              />
+            ) : (
+              <p>{message}</p>
+            )}
+            {!!reviewerComment && (
+              <div className="comment-container">
+                <Icon name="comment alternate outline" color="grey" />
+                <p className="secondary">{reviewerComment}</p>
+              </div>
+            )}
+          </>
         ) : (
-          <p>{message}</p>
-        )}
-        {!!reviewerComment && (
-          <div className="comment-container">
-            <Icon name="comment alternate outline" color="grey" />
-            <p className="secondary">{reviewerComment}</p>
-          </div>
+          <p>
+            <em>{strings.REVIEW_HISTORY_NO_RESPONSE}</em>
+          </p>
         )}
```

Comment:
Not sure why we have this change added here. Isn't this already in `develop` @CarlosNZ ?

Reply:
No, I did that fix as part of this branch cos I thought it would be interacting with some of the other changes we were making. Perhaps I shouldn't have in hindsight.

Reply:
It's mostly in the back-end though.

Reply:
Thanks! I'll add to the list of changes in the description. Have updated it now with all that I noticed is included